### PR TITLE
[thud] white-space compliant append

### PR DESCRIPTION
**Pull request by IniterWorker**
# Starting with White-Space

## Regex Used
```
sed -i  -e "s/\(.*_append.* = \"\)\([^ $,+-]\)/\1 \2/"
```

Used to complete #58

### CODE REVIEW

**Comment by fbertux** [comment]
Hi @IniterWorker 

You don't need to add extra space when using `+=` operator, this is needed only with `_append`. See the [here](https://www.yoctoproject.org/docs/1.6/bitbake-user-manual/bitbake-user-manual.html#appending-and-prepending).

Thanks!

**Comment by IniterWorker** [comment]
Hi @fbertux,

I'm just reading about it.

3.1.6. Appending (+=) and prepending (=+) With Spaces

I will change my regex.

**Comment by otavio** [comment]
@IniterWorker when you do that, please update all PR and do a force-push so we keep a clean commit history.

**Comment by IniterWorker** [comment]
@otavio this is precisely what I did.
I have a question for you on one change.

```
diff --git a/recipes-graphics/mesa/mesa_%.bbappend b/recipes-graphics/mesa/mesa_%.bbappend
index eaa94eef..ea2c7c55 100644
--- a/recipes-graphics/mesa/mesa_%.bbappend
+++ b/recipes-graphics/mesa/mesa_%.bbappend
@@ -15,7 +15,7 @@ python () {
 
 # Enable Etnaviv support
 PACKAGECONFIG_append_use-mainline-bsp = " gallium"
-GALLIUMDRIVERS_append_use-mainline-bsp = ",etnaviv,imx"
+GALLIUMDRIVERS_append_use-mainline-bsp = " ,etnaviv,imx"
```

Is it correct or I need to change my regex to keep it without white-space the ','?

**Comment by otavio** [comment]
This one should not be changed. It should remain as it is.

File: recipes-bsp/imx-mkimage/imx-boot_0.2.bb [hunk 1]
```diff
@@ -40,7 +40,7 @@ ATF_MACHINE_NAME_mx8qm = "bl31-imx8qm.bin"
 ATF_MACHINE_NAME_mx8qxp = "bl31-imx8qxp.bin"
 ATF_MACHINE_NAME_mx8mq = "bl31-imx8mq.bin"
 ATF_MACHINE_NAME_mx8mm = "bl31-imx8mm.bin"
-ATF_MACHINE_NAME_append = "${@bb.utils.contains('COMBINED_FEATURES', 'optee', '-optee', '', d)}"
+ATF_MACHINE_NAME_append = " ${@bb.utils.contains('COMBINED_FEATURES', 'optee', '-optee', '', d)}"
```
Comment (otavio) [inline recipes-bsp/imx-mkimage/imx-boot_0.2.bb]: Wrong.

File: recipes-bsp/u-boot/u-boot-qoriq-common_2018.09.inc [hunk 2]
```diff
@@ -15,5 +15,5 @@ SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-components/u-boot;no
 SRCREV= "c08de7c3c92d45a824500ad936f4d72465c181ec"
 
 S = "${WORKDIR}/git"
-PV_append = "+fslgit"
+PV_append = " +fslgit"
```
Comment (otavio) [inline recipes-bsp/u-boot/u-boot-qoriq-common_2018.09.inc]: Wrong.

File: recipes-graphics/mesa/mesa_%.bbappend [hunk 3]
```diff
@@ -16,7 +16,7 @@ python () {
 
 # Enable Etnaviv support
 PACKAGECONFIG_append_use-mainline-bsp = " gallium"
-GALLIUMDRIVERS_append_use-mainline-bsp = ",etnaviv,imx"
+GALLIUMDRIVERS_append_use-mainline-bsp = " ,etnaviv,imx"
```
Comment (otavio) [inline recipes-graphics/mesa/mesa_%.bbappend]: Wrong.

**Comment by IniterWorker** [comment]
I'm working on it.

**Comment by IniterWorker** [comment]
@otavio the force push removed the review link. Are you okay with the last modification?

**Pull request merged**